packages/features/src/index.ts

```diff
@@ -89,6 +89,10 @@ export class Service {
   ): Promise<IGetFeaturesResponse> {
     const requestParams: IRequestParams = Object.assign({}, options.params);
 
+    if (options.crs) {
+      requestParams.crs = stringifyCrs(options.crs);
+    }
```

Comment:
Can you update the unit test to verify the `crs` string would exist if the option is given?

Reply:
I thought I had removed this section. Sry for the confusion. That should actually go into a separate PR. You can decide whether I should  fill a separate PR for the `crs` issue or if I should use this PR to resolve the issue.

Reply:
Using a second PR is more proper. Once you remove these lines, I will ungate this PR.

Reply:
I have removed the crs lines.